src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,26 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import './PostList.scss';
+import { PostInfo } from '../PostInfo';
+
+import { Post } from '../../types/post';
+import { Comment } from '../../types/comment';
+import { User } from '../../types/user';
+
+type Props = {
+  posts: Post[],
+  comments: Comment[],
+  users: User[],
+};
+
+export const PostList: React.FC<Props> = ({ posts, comments, users }) => (
+  <div className="PostList">
+    {posts.map((post: Post) => (
+      <PostInfo
```

Comment:
It is recommended to add a unique 'key' prop when mapping over an array to create a list of elements. This will help React to identify which items have changed, are added, or are removed, and will result in a better performance of the app.